README.md

```diff
@@ -73,19 +141,58 @@ That is the explicit way to push. You can also implicitly push your code by runn
 ```bash
 git push
 ```
-This will push your code up to the remote repo/branch you're tracking. The first time you push code up to a newly-added remote repository, use the `-u` flag to tell Git to track the remote repository: `git push -u origin master`. For every subsequent push, plain old `git push` will suffice.
+This will push your code up to the remote repo/branch you're tracking. The
+first time you push code up to a newly-added remote repository, use the `-u`
+flag to tell Git to track the remote repository: `git push -u origin master`.
+For every subsequent push, plain old `git push` will suffice.
 
 [For more details, check out the GitHub guide on pushing.](https://help.github.com/articles/pushing-to-a-remote/)
 
-### `git pull`
 
-As we collaborate with other people, inevitably they will push some code. The only problem is that now the code on our machine (our local repo) is out of sync with the remote repo. To remedy this, we must pull down the new code from the remote repo to our local. No surprise here. To do this, run:
+##  Use `git pull` to Retrieve Code from the Remote Repo
+
+As we collaborate with other people, inevitably, they will push some code. The
+only problem is that once this happens, the code on our machine (our local
+repo) will be out of sync with the remote repo. To remedy this, we must pull
+down the new code from the remote repo to our local. No surprise here. To do
+this, run:
 
 ```bash
 git pull
 ```
 
-Again, we can also do this explicitly if need be by adding the remote name and branch as arguments: `git pull origin master`.
+In our case, running `git pull` should return the message `Already up-to-date.`,
+since our remote hasn't been modified.  If there was new content, the terminal
+will show that it is unpacking and updating, then list out any files that have
+been updated.
+
+Again, we can also do this explicitly if need be by adding the remote name and
+branch as arguments: `git pull origin master`.
+
+[For more details, check out the GitHub guide on pulling.](https://help.github.com/articles/fetching-a-remote/)
+
+## Use Github to Make Direct Edits
+
+Say you liked your README, but you noticed a minor typo. Let's fix it directly on GitHub.
+
+1. Navigate to your remote repository on [GitHub.com](https://github.com/), e.g., https://github.com/username-here/repository-name-here.
+2. Click on your README file.
+3. At the top of the file, you'll notice a pencil icon (![GitHub pencil](http://i.imgur.com/J3HiLhO.png)). Clicking this will allow us to edit the file.
+4. Make some changes to your README.
+5. Commit them by clicking the "Commit changes" button at the bottom of the page.
+
+Perfect! But now the code on our machine (in our local repo) is out of sync with the remote repo. To remedy this, we must `pull` down the code to our local repo. To do so, run:
+
+```bash
+git pull
+``` 
+
+## Conclusion
+
+Being able to add and update git remotes allows you to be able to get repos started up
```

Comment:
...lets you back up your local repository to a remote server. 

Let's be precise about the benefit